src/spikeinterface/core/testing.py

```diff
@@ -88,6 +90,19 @@ def check_sortings_equal(
             train1 = np.sort(SX1.get_unit_spike_train(id, segment_index=segment_idx))
             train2 = np.sort(SX2.get_unit_spike_train(id, segment_index=segment_idx))
             assert np.array_equal(train1, train2)
+            train1 = np.sort(SX1.get_unit_spike_train(id, segment_index=segment_idx, start_frame=30))
```

Comment:
Can you add a comment on why those were included so the future person who sees this can make sense of this? As in, "it is important to tests this because ... ' 


 Maybe testing for these properties could be a different function with a meaningful name that is called from within `check_sortings_equal` or from somewhere else.

Reply:
done